### ISSUE TITLE
K3OS AWS AMI Fails to boot

### ISSUE THREAD

**Issue by canadiannomad** [rancher/k3os#540]
**Version (k3OS / kernel)**
Followed steps from v0.11.0-rc1, k3os/package/packer/aws/README.md
But gives me v0.10.2 I believe.

**Architecture**
arm64 - a1.large

**Describe the bug**
When I boot the created ami, it fails to initialize, I'm unable to SSH in.  When I am able to get the System Log via AWS Console, I see the following:

```
++ TARGET=/run/k3os/target
++ source /usr/lib/os-release
+++ NAME=k3OS
+++ VERSION='k3OS v0.10.2'
+++ ID=k3os
+++ ID_LIKE=alpine
+++ PRETTY_NAME='k3OS v0.10.2'
+++ VERSION_ID=v0.10.2
+++ HOME_URL=https://k3os.io/
+++ SUPPORT_URL=https://k3os.io/
+++ BUG_REPORT_URL=https://github.com/rancher/k3os/issues
+++ ISO_URL=https://github.com/rancher/k3os/releases/download/v0.10.2/k3os-arm64.iso
++ setup_mounts
++ mkdir -p /run/k3os/target
++ mount -L K3OS_STATE /run/k3os/target
[    4.099329] EXT4-fs (nvme0n1p1): recovery complete
[    4.103783] EXT4-fs (nvme0n1p1): mounted filesystem with ordered data mode. Opts: (null)
++ '[' -e /run/k3os/target/k3os/system/growpart ']'
++ setup_k3os
++ '[' -e /run/k3os/target/k3os/system/k3os/current/k3os ']'
++ return 0
++ setup_kernel_squashfs
+++ uname -r
++ KER_SRC=/.base/k3os/system/kernel/5.0.0-43-generic/kernel.squashfs
+++ uname -r
++ KER_DEST=/run/k3os/target/k3os/system/kernel/5.0.0-43-generic/kernel.squashfs
++ '[' -e /.base/k3os/system/kernel/5.0.0-43-generic/kernel.squashfs ']'
++ setup_init
++ '[' -e /run/k3os/target/sbin/init ']'
++ mkdir -p /run/k3os/target/sbin
mkdir: cannot create directory ‘/run/k3os/target/sbin’: File exists
+ rescue
+ echo ERROR 'Something went wrong, run with cmdline k3os.debug for more logging'
ERROR Something went wrong, run with cmdline k3os.debug for more logging
+ echo Dropping to shell
Dropping to shell
+ exec bash
bash: cannot set terminal process group (-1): Not a tty
bash: no job control in this shell
```

**To Reproduce**
Follow steps from v0.11.0-rc1, k3os/package/packer/aws/README.md
Launch created AMI.
Check System Logs when it fails to initialize.

**Expected behavior**
Should boot at least to a point where I can debug the config.yaml

**Actual behavior**
AMI fails to boot, AWS Console reports "1/2 checks passed" in status, and all ports are closed.

**Additional context**
Attempted to change the mkdir statement via packer, but it seems that had no effect.

**Comment by dweomer** [comment]
Possible fixed by #551, validating.

**Comment by dweomer** [comment]
Hmm, unrelated to #551 but you do need to edit the packer template to ensure it has the correct ISO URL:
```patch
diff --git a/package/packer/aws/template-arm64.json b/package/packer/aws/template-arm64.json
index 476c6fb..affcc5a 100644
--- a/package/packer/aws/template-arm64.json
+++ b/package/packer/aws/template-arm64.json
@@ -55,9 +55,9 @@
   "variables": {
     "aws_access_key": "{{env `AWS_ACCESS_KEY_ID`}}",
     "aws_secret_key": "{{env `AWS_SECRET_ACCESS_KEY`}}",
-    "iso_url": "https://github.com/rancher/k3os/releases/download/v0.10.2/k3os-arm64.iso",
-    "k3os_version": "v0.10.2",
-    "region": "us-east-1"
+    "iso_url": "https://github.com/rancher/k3os/releases/download/v0.11.0-rc2/k3os-arm64.iso",
+    "k3os_version": "v0.11.0-rc2",
+    "region": "us-west-2"
   }
 }
 
diff --git a/package/packer/aws/template.json b/package/packer/aws/template.json
index 1c7c704..f1895b6 100644
--- a/package/packer/aws/template.json
+++ b/package/packer/aws/template.json
@@ -55,9 +55,9 @@
   "variables": {
     "aws_access_key": "{{env `AWS_ACCESS_KEY_ID`}}",
     "aws_secret_key": "{{env `AWS_SECRET_ACCESS_KEY`}}",
-    "iso_url": "https://github.com/rancher/k3os/releases/download/v0.10.2/k3os-amd64.iso",
-    "k3os_version": "v0.10.2",
-    "region": "us-east-1"
+    "iso_url": "https://github.com/rancher/k3os/releases/download/v0.11.0-rc2/k3os-amd64.iso",
+    "k3os_version": "v0.11.0-rc2",
+    "region": "us-west-2"
   }
 }
```

```
Linux ip-172-31-35-34.us-west-2.compute.internal 5.4.0-37-generic #41 SMP Fri Jun 19 23:12:44 UTC 2020 aarch64 GNU/Linux
ip-172-31-35-34 [~]$ cat /etc/os-release 
NAME="k3OS"
VERSION="k3OS v0.11.0-rc2"
ID=k3os
ID_LIKE=alpine
PRETTY_NAME="k3OS v0.11.0-rc2"
VERSION_ID="v0.11.0-rc2"
HOME_URL="https://k3os.io/"
SUPPORT_URL="https://k3os.io/"
BUG_REPORT_URL="https://github.com/rancher/k3os/issues"
ISO_URL="https://github.com/rancher/k3os/releases/download/v0.11.0-rc2/k3os-arm64.iso"
ip-172-31-35-34 [~]$ 
```
